flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/entries/ExecutionEntry.java

```diff
@@ -108,6 +108,10 @@
 
 	public static final String EXECUTION_CURRENT_DATABASE = "current-database";
 
+	private static final String EXECUTION_INTERACTIVE_VIEW_ENABLED = "interactive-view.enabled";
+
+	private static final String EXECUTION_MAX_QUERY_TIMEOUT_MS = "max-query-timeout-ms";
```

Comment:
When do you think a user wants to set this? If the connection to the cluster is bad?

Reply:
It offers an overall protection for the user or downstream projects submitting a long-running query(e.g., select * from kafakTable). IMO for DQL cases in a streaming scenario, user always expects the result is finalizable(He/She can not stare at the screen all of the day). For user, he can use CTRL+C signal. For downstream project, a timeout should be better.